Sources/Stencil/Template.swift

```diff
@@ -41,19 +40,7 @@ open class Template: ExpressibleByStringLiteral {
       throw NSError(domain: NSCocoaErrorDomain, code: NSFileNoSuchFileError, userInfo: nil)
     }
 
-    try self.init(URL: url)
-  }
-
-  /// Create a template with a file found at the given URL
-  @available(*, deprecated, message: "Use Environment/FileSystemLoader instead")
```

Comment:
So if we're dropping these because we no longer have the `Path` type, I'm assuming it'll be at least a "minor" version bump (considering we're still on 0.x.x)

Reply:
Yeah, that seems fair.